extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicExporter.java

```diff
@@ -96,8 +96,14 @@ public GoogleMusicExporter(
   public ExportResult<MusicContainerResource> export(
       UUID jobId, TokensAndUrlAuthData authData, Optional<ExportInformation> exportInformation)
       throws IOException, InvalidTokenException, PermissionDeniedException {
-
-    if (exportInformation.get().getContainerResource() instanceof IdOnlyContainerResource) {
+    // Local demo-server test usage. Start transfer job without ExportInformation.
```

Comment:
TODO to remove this? or you mean to leave this here on purpose as documentation/reference?

Reply:
Added TODO to remove the testing code in the following commit.